torch/_inductor/codecache.py

```diff
@@ -119,15 +129,15 @@ def cpp_wrapper_cache_dir(name):
 class CacheBase:
     @staticmethod
     @functools.lru_cache(None)
-    def get_system():
+    def get_system() -> Dict[str, Any]:
```

Comment:
So there are several places throughout where I've used `Any` and would have liked to avoid. But a lot of these caching utilities generally seem to be pretty liberal about the types of objects, e.g., I was seeing instances of both objects and primitive types. So I kinda gave up wherever you see `Any`. Let me know if anywhere looks especially lazy and/or you'd prefer to be stricter even if it means a ridiculous Union of possible types.

Reply:
Should it be Any or just Python object? Main difference is Any can be a None type too etc.